x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/curation/curation_logic.test.ts

```diff
@@ -60,7 +62,7 @@ describe('CurationLogic', () => {
 
   describe('actions', () => {
     describe('onCurationLoad', () => {
-      it('should set curation state & dataLoading to false', () => {
+      it('should set curation state & all loading states to false', () => {
```

Comment:
A quick note on this - various Curation sections (namely: queries, promoted documents, hidden documents, and organic documents) will each have their own loading state. This is for the sake of UI/UX; so we're still showing responsive reactions but not blanking out the entire page every time we make a single XHR call.

The queries loading state is shown in the PR loading description, and the promoted/hidden loading state will use the new LoadingOverlay component: https://github.com/elastic/kibana/pull/94251#discussion_r590886936